packages/gatsby/content/advanced/questions-and-answers.md

```diff
@@ -38,44 +38,50 @@ Most projects will only face those three problems, which can all be fixed in a g
 
 ## Which files should be gitignored?
 
-If you're using Zero-Installs:
+Most commonly, the following patterns should be ignored:
 
 ```gitignore
 .yarn/*
-!.yarn/cache
 !.yarn/releases
 !.yarn/plugins
 !.yarn/sdks
 !.yarn/versions
-```
 
-If you're not using Zero-Installs:
-
-```gitignore
-.yarn/*
-!.yarn/releases
-!.yarn/plugins
-!.yarn/sdks
-!.yarn/versions
+# Include to skip downloading packages when running an install:
+!.yarn/cache
 .pnp.*
+
+# (!) Only include if using Zero-Installs:
+!.yarn/unplugged 
+!.yarn/build-state.yml
```

Comment:
They can be machine specific and shouldn't be commited though :worried:

Reply:
The paragraph below mentions committing them in certain circumstances. Should that be changed as well?

https://github.com/yarnpkg/berry/pull/1953/files#diff-9d1f5f060e8d47d00e2dcbddc2201bdfR65